# Created by Chinmay Kamat <chinmaykamat@gmail.com>
# http://mytechrants.wordpress.com/2009/11/26/wget-add-progressbar-nv/
# Adapted by Gabriel B. Casella <gbc921@gmail.com>
--- wget.orig/src/main.c   2009-09-22 08:33:11.000000000 +0530
+++ wget/src/main.c    2009-11-26 12:21:15.156859000 +0530
@@ -1140,7 +1140,7 @@
 
   /* Initialize progress.  Have to do this after the options are
      processed so we know where the log file is.  */
-  if (opt.verbose)
+  if (!opt.quiet)
     set_progress_implementation (opt.progress_type);
 
   /* Fill in the arguments.  */
 
--- wget.orig/src/progress.c   2009-09-22 08:19:32.000000000 +0530
+++ wget/src/progress.c    2009-11-26 12:53:11.309295000 +0530
@@ -291,7 +291,7 @@
          been retrieved.  12.8% will round to 12% because the 13% mark
          has not yet been reached.  100% is only shown when done.  */
       int percentage = 100.0 * bytes_displayed / dp->total_length;
-      logprintf (LOG_VERBOSE, "%3d%%", percentage);
+      logprintf (LOG_NOTQUIET, "%3d%%", percentage);
     }
 
   {
@@ -325,7 +325,7 @@
           wgint bytes_sofar = bytes_displayed - dp->initial_length;
           double eta = dltime * bytes_remaining / bytes_sofar;
           if (eta < INT_MAX - 1)
-            logprintf (LOG_VERBOSE, " %s",
+            logprintf (LOG_NOTQUIET, " %s",
                        eta_to_human_short ((int) (eta + 0.5), true));
         }
     }
@@ -333,10 +333,10 @@
     {
       /* When done, print the total download time */
       if (dltime >= 10)
-        logprintf (LOG_VERBOSE, "=%s",
+        logprintf (LOG_NOTQUIET, "=%s",
                    eta_to_human_short ((int) (dltime + 0.5), true));
       else
-        logprintf (LOG_VERBOSE, "=%ss", print_decimal (dltime));
+        logprintf (LOG_NOTQUIET, "=%ss", print_decimal (dltime));
     }
 }
 
@@ -652,7 +652,7 @@
   create_image (bp, dltime, true);
   display_image (bp->buffer);
 
-  logputs (LOG_VERBOSE, "\n\n");
+  logputs (LOG_NOTQUIET, "\n\n");
 
   xfree (bp->buffer);
   xfree (bp);
@@ -1071,8 +1071,8 @@
 display_image (char *buf)
 {
   bool old = log_set_save_context (false);
-  logputs (LOG_VERBOSE, "\r");
-  logputs (LOG_VERBOSE, buf);
+  logputs (LOG_NOTQUIET, "\r");
+  logputs (LOG_NOTQUIET, buf);
   log_set_save_context (old);
 }
 
--- wget.orig/src/retr.c   2009-09-04 22:01:54.000000000 +0530
+++ wget/src/retr.c    2009-11-26 12:21:15.167860000 +0530
@@ -234,7 +234,7 @@
   if (flags & rb_skip_startpos)
     skip = startpos;
 
-  if (opt.verbose)
+  if (!opt.quiet)
     {
       /* If we're skipping STARTPOS bytes, pass 0 as the INITIAL
          argument to progress_create because the indicator doesn't
